activemodel/lib/active_model/type/collection.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module ActiveModel
+  module Type
+    # Attribute type for floating point numeric values. It is registered under
```

Comment:
Should this be something like:
```suggestion
    # Attribute type for a collection values. It is registered under
```

Reply:
Oh, yes, of course! 🙈 
Let me change this. Thanks!